README.md

```diff
@@ -7,9 +7,85 @@ Table of Contents
 =================
 
 * [Name](#name)
+* [Synopsis](#synopsis)
 * [Description](#description)
 * [Copyright & License](#copyright--license)
 
+Synopsis
+========
+
+Basic usage
+
+```
+
+# set variable in ob file scope.
+foo = bar;  we create a variable foo which value is bar. variable in ob is string 
+replace , op will render variable like some template engine.
+
+# comment in ob file.
+#foo = bar; ob use # as comment syntax.
+
+# block in ob file.
+foo {
+    command1; 
+    command2; 
+}
+
+Block is basic execute unit.we use op compile out pl with make keyword to execute wanted unit.
```

Comment:
It's not really an execution unit since the statements in the block are also declarative per se (just like `nginx.conf`).

You need to put a space after the full stop.

Also the last sentence is unreadable. I think you mean the following:

```
we use the opsboy compiler (with the `make` command-line argument) to generate standalone Perl script from the whole opsboy specification (not just a single rule specification).
```